run_linux.go

```diff
@@ -1736,6 +1738,13 @@ func (b *Builder) runSetupVolumeMounts(mountLabel string, volumeMounts []string,
 			}
 			return overlayMount, err
 		}
+		if foundU {
+			if err := filepath.Walk(host, func(filePath string, f os.FileInfo, err error) error {
```

Comment:
In the SELinux relabel, we check to make sure that the user is not "relabeling" a system path, Since this could cause major issues.

https://github.com/opencontainers/selinux/blob/master/go-selinux/label/label_selinux.go#L112

Do we want to do the same here?

Reply:
I can check the current uid, gid before doing the chown operation. I'll add that. Thanks